app/tests/test_generic_api.py

```diff
@@ -404,19 +404,19 @@ class ApiCacheHeaderTestCase(StacBaseTestCase):
 
     @classmethod
     @mock_s3_asset_file
-    def setUpTestData(cls):
```

Comment:
Why changing it from `setUpTestData` to `setUp` ?

NOTE: is this is really required the `@classmethod` needs also to be removed.